validator/src/network/tendermint/mod.rs

```diff
@@ -57,12 +57,12 @@ impl abci::Application for Abci {
 
     fn commit(&mut self, _req: &RequestCommit) -> ResponseCommit {
         // Create the response
-        let mut resp = ResponseCommit::new();
+        let mut response = ResponseCommit::new();
         // Convert count to bits
         let mut buf = [0; 8];
         BigEndian::write_u64(&mut buf, self.count);
```

Comment:
There are nice helper functions to achieve that functionality: https://doc.rust-lang.org/std/primitive.u64.html#method.to_be_bytes